localstack/services/stores.py

```diff
@@ -200,7 +206,7 @@ def reset(self):
             self.clear()
 
 
-class AccountRegionBundle(dict):
+class AccountRegionBundle(dict, Generic[BaseStoreType]):
```

Comment:
nit: Perhaps we should also return the generic type `__getitem__(..) -> RegionBundle[BaseStoreType]:` in line 225 below.